diff --git a/gradle/libs.versions.toml b/gradle/libs.versions.toml
index 52e9a741e922..4631f58c521d 100644
--- a/gradle/libs.versions.toml
+++ b/gradle/libs.versions.toml
@@ -50,14 +50,9 @@ work = "2.10.5"
 
 # Google other
 accompanist = "0.37.3"
-firebase-messaging = "25.0.1"
 installreferrer = "2.2"
 material = "1.13.0"
 osslicenses-plugin = "0.10.7"
-play-review = "2.0.2"
-play-services-ads-id = "18.2.0"
-play-services-base = "18.9.0"
-play-services-fido = "21.2.0"
 protobuf = "4.32.1" # Keep Protobuf in sync with the version used by AppServices.
 
 # Gradle plugins
@@ -201,15 +196,9 @@ androidx-compose-ui-tooling-preview = { group = "androidx.compose.ui", name = "u
 
 # Google other
 accompanist-drawablepainter = { group = "com.google.accompanist", name = "accompanist-drawablepainter", version.ref = "accompanist" }
-firebase-messaging = { group = "com.google.firebase", name = "firebase-messaging", version.ref = "firebase-messaging" }
 installreferrer = { group = "com.android.installreferrer", name = "installreferrer", version.ref = "installreferrer" }
 google-material = { group = "com.google.android.material", name = "material", version.ref = "material" }
 osslicenses-plugin = { group = "com.google.android.gms", name = "oss-licenses-plugin", version.ref = "osslicenses-plugin" }
-play-review = { group = "com.google.android.play", name = "review", version.ref = "play-review" }
-play-review-ktx = { group = "com.google.android.play", name = "review-ktx", version.ref = "play-review" }
-play-services-ads-id = { group = "com.google.android.gms", name = "play-services-ads-identifier", version.ref = "play-services-ads-id" }
-play-services-base = { group = "com.google.android.gms", name = "play-services-base", version.ref = "play-services-base" }
-play-services-fido = { group = "com.google.android.gms", name = "play-services-fido", version.ref = "play-services-fido" }
 protobuf-compiler = { group = "com.google.protobuf", name = "protoc", version.ref = "protobuf" }
 protobuf-javalite = { group = "com.google.protobuf", name = "protobuf-javalite", version.ref = "protobuf" }
 
diff --git a/mobile/android/android-components/.buildconfig.yml b/mobile/android/android-components/.buildconfig.yml
index f9711dc715e7..c03556c7c7c9 100644
--- a/mobile/android/android-components/.buildconfig.yml
+++ b/mobile/android/android-components/.buildconfig.yml
@@ -1696,17 +1696,6 @@ projects:
     - components:support-base
     - components:support-test
     - components:tooling-lint
-  components:lib-push-firebase:
-    description: An implementation of concept-push for the Firebase Message Service.
-    path: components/lib/push-firebase
-    publish: true
-    upstream_dependencies:
-    - components:concept-base
-    - components:concept-fetch
-    - components:concept-push
-    - components:support-base
-    - components:support-test
-    - components:tooling-lint
   components:lib-state:
     description: A library for maintaining application state.
     path: components/lib/state
diff --git a/mobile/android/fenix/.buildconfig.yml b/mobile/android/fenix/.buildconfig.yml
index 1753bcbd86ff..fad6fde5c2a0 100644
--- a/mobile/android/fenix/.buildconfig.yml
+++ b/mobile/android/fenix/.buildconfig.yml
@@ -65,7 +65,6 @@ projects:
     - components:lib-crash-sentry
     - components:lib-dataprotect
     - components:lib-publicsuffixlist
-    - components:lib-push-firebase
     - components:lib-state
     - components:service-digitalassetlinks
     - components:service-firefox-accounts
diff --git a/mobile/android/fenix/app/build.gradle b/mobile/android/fenix/app/build.gradle
index af2ddca5a621..5d0b5295f5d1 100644
--- a/mobile/android/fenix/app/build.gradle
+++ b/mobile/android/fenix/app/build.gradle
@@ -637,12 +637,10 @@ dependencies {
     implementation project(':components:lib-crash')
     implementation project(':components:lib-crash-sentry')
     implementation project(':components:lib-dataprotect')
-    implementation project(':components:lib-push-firebase')
     implementation project(':components:lib-state')
 
     implementation ComponentsDependencies.mozilla_appservices_syncmanager
     implementation libs.accompanist.drawablepainter
-    implementation libs.adjust
     implementation libs.androidx.activity
     implementation libs.androidx.activity.ktx
     implementation libs.androidx.annotation
@@ -688,16 +686,14 @@ dependencies {
     implementation libs.androidx.viewpager2
     implementation libs.androidx.work.runtime
     implementation libs.google.material
-    implementation libs.installreferrer
     implementation libs.kotlinx.coroutines
     implementation libs.kotlinx.serialization.json
     implementation libs.mozilla.glean
-    implementation libs.play.review
-    implementation libs.play.review.ktx
-    implementation libs.play.services.ads.id
     implementation libs.protobuf.javalite
     implementation libs.sentry
 
+    implementation 'org.microg.gms:play-services-ads-identifier:0.3.10.250932'
+
     debugImplementation libs.androidx.compose.ui.tooling
     debugImplementation libs.leakcanary
 
diff --git a/mobile/android/fenix/app/proguard-rules.pro b/mobile/android/fenix/app/proguard-rules.pro
index 39a6df6fb62e..8eb0cab2d939 100644
--- a/mobile/android/fenix/app/proguard-rules.pro
+++ b/mobile/android/fenix/app/proguard-rules.pro
@@ -28,3 +28,11 @@
 
 # Keep code generated from Glean Metrics
 -keep class org.mozilla.fenix.GleanMetrics.** {  *; }
+
+####################################################################################################
+# Checker Framework
+####################################################################################################
+
+-dontwarn org.checkerframework.checker.nullness.qual.EnsuresNonNull
+-dontwarn org.checkerframework.checker.nullness.qual.EnsuresNonNullIf
+-dontwarn org.checkerframework.checker.nullness.qual.RequiresNonNull
diff --git a/mobile/android/fenix/app/src/main/java/com/adjust/sdk/Adjust.java b/mobile/android/fenix/app/src/main/java/com/adjust/sdk/Adjust.java
new file mode 100644
index 000000000000..d377e98bbf05
--- /dev/null
+++ b/mobile/android/fenix/app/src/main/java/com/adjust/sdk/Adjust.java
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2012-2017 adjust GmbH,
+ * http://www.adjust.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining
+ * a copy of this software and associated documentation files (the
+ * "Software"), to deal in the Software without restriction, including
+ * without limitation the rights to use, copy, modify, merge, publish,
+ * distribute, sublicense, and/or sell copies of the Software, and to
+ * permit persons to whom the Software is furnished to do so, subject to
+ * the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be
+ * included in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+ * LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+ * WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package com.adjust.sdk;
+
+import android.content.Context;
+
+public class Adjust {
+    public static void initSdk(AdjustConfig adjustConfig) {
+    }
+
+    public static void trackEvent(AdjustEvent event) {
+    }
+
+    public static void onResume() {
+    }
+
+    public static void onPause() {
+    }
+
+    public static void enable() {
+    }
+
+    public static void disable() {
+    }
+
+    public static void gdprForgetMe(final Context context) {
+    }
+}
diff --git a/mobile/android/fenix/app/src/main/java/com/adjust/sdk/AdjustAttribution.java b/mobile/android/fenix/app/src/main/java/com/adjust/sdk/AdjustAttribution.java
new file mode 100644
index 000000000000..ab6b3badbdb1
--- /dev/null
+++ b/mobile/android/fenix/app/src/main/java/com/adjust/sdk/AdjustAttribution.java
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2012-2017 adjust GmbH,
+ * http://www.adjust.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining
+ * a copy of this software and associated documentation files (the
+ * "Software"), to deal in the Software without restriction, including
+ * without limitation the rights to use, copy, modify, merge, publish,
+ * distribute, sublicense, and/or sell copies of the Software, and to
+ * permit persons to whom the Software is furnished to do so, subject to
+ * the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be
+ * included in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+ * LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+ * WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package com.adjust.sdk;
+
+import java.io.Serializable;
+
+public class AdjustAttribution implements Serializable {
+    public String network;
+    public String campaign;
+    public String adgroup;
+    public String creative;
+
+    @Override
+    public boolean equals(Object other) {
+        return false;
+    }
+
+    @Override
+    public int hashCode() {
+        return 0;
+    }
+
+    @Override
+    public String toString() {
+        return "";
+    }
+}
diff --git a/mobile/android/fenix/app/src/main/java/com/adjust/sdk/AdjustConfig.java b/mobile/android/fenix/app/src/main/java/com/adjust/sdk/AdjustConfig.java
new file mode 100644
index 000000000000..f563917d2070
--- /dev/null
+++ b/mobile/android/fenix/app/src/main/java/com/adjust/sdk/AdjustConfig.java
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2012-2017 adjust GmbH,
+ * http://www.adjust.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining
+ * a copy of this software and associated documentation files (the
+ * "Software"), to deal in the Software without restriction, including
+ * without limitation the rights to use, copy, modify, merge, publish,
+ * distribute, sublicense, and/or sell copies of the Software, and to
+ * permit persons to whom the Software is furnished to do so, subject to
+ * the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be
+ * included in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+ * LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+ * WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package com.adjust.sdk;
+
+import android.content.Context;
+
+import java.util.List;
+
+public class AdjustConfig {
+    public static final String ENVIRONMENT_SANDBOX = "sandbox";
+    public static final String ENVIRONMENT_PRODUCTION = "production";
+
+    public AdjustConfig(Context context, String appToken, String environment) {
+    }
+
+    public AdjustConfig(Context context, String appToken, String environment, boolean allowSuppressLogLevel) {
+    }
+
+    public void setOnAttributionChangedListener(OnAttributionChangedListener onAttributionChangedListener) {
+    }
+
+    public void enablePreinstallTracking() {
+    }
+
+    public void setLogLevel(LogLevel logLevel) {
+    }
+}
diff --git a/mobile/android/fenix/app/src/main/java/com/adjust/sdk/AdjustEvent.java b/mobile/android/fenix/app/src/main/java/com/adjust/sdk/AdjustEvent.java
new file mode 100644
index 000000000000..0b4d80ed4b2c
--- /dev/null
+++ b/mobile/android/fenix/app/src/main/java/com/adjust/sdk/AdjustEvent.java
@@ -0,0 +1,31 @@
+package com.adjust.sdk;
+
+/**
+ * Created by pfms on 05/11/14.
+ */
+public class AdjustEvent {
+
+    public AdjustEvent(String eventToken) {
+    }
+
+    public void setRevenue(double revenue, String currency) {
+    }
+
+    public void addCallbackParameter(String key, String value) {
+    }
+
+    public void addPartnerParameter(String key, String value) {
+    }
+
+    public void setOrderId(String orderId) {
+    }
+
+    public void setCallbackId(String callbackId) {
+    }
+
+    public void setProductId(String productId) {
+    }
+
+    public void setPurchaseToken(String purchaseToken) {
+    }
+}
diff --git a/mobile/android/fenix/app/src/main/java/com/adjust/sdk/Constants.java b/mobile/android/fenix/app/src/main/java/com/adjust/sdk/Constants.java
new file mode 100644
index 000000000000..bfaeee26e8e1
--- /dev/null
+++ b/mobile/android/fenix/app/src/main/java/com/adjust/sdk/Constants.java
@@ -0,0 +1,18 @@
+//
+//  Constants.java
+//  Adjust
+//
+//  Created by keyboardsurfer on 2013-11-08.
+//  Copyright (c) 2012-2014 adjust GmbH. All rights reserved.
+//  See the file MIT-LICENSE for copying permission.
+//
+
+package com.adjust.sdk;
+
+/**
+ * @author keyboardsurfer
+ * @since 8.11.13
+ */
+public interface Constants {
+    String ADJUST_PREINSTALL_SYSTEM_PROPERTY_PATH = "adjust.preinstall.path";
+}
diff --git a/mobile/android/fenix/app/src/main/java/com/adjust/sdk/LogLevel.java b/mobile/android/fenix/app/src/main/java/com/adjust/sdk/LogLevel.java
new file mode 100644
index 000000000000..f4ef17f4c492
--- /dev/null
+++ b/mobile/android/fenix/app/src/main/java/com/adjust/sdk/LogLevel.java
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2012-2017 adjust GmbH,
+ * http://www.adjust.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining
+ * a copy of this software and associated documentation files (the
+ * "Software"), to deal in the Software without restriction, including
+ * without limitation the rights to use, copy, modify, merge, publish,
+ * distribute, sublicense, and/or sell copies of the Software, and to
+ * permit persons to whom the Software is furnished to do so, subject to
+ * the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be
+ * included in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+ * LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+ * WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package com.adjust.sdk;
+
+import android.util.Log;
+
+/**
+ * Created by pfms on 11/03/15.
+ */
+public enum LogLevel {
+    VERBOSE(Log.VERBOSE), DEBUG(Log.DEBUG), INFO(Log.INFO), WARN(Log.WARN), ERROR(Log.ERROR), ASSERT(Log.ASSERT), SUPPRESS(8);
+    final int androidLogLevel;
+
+    LogLevel(final int androidLogLevel) {
+        this.androidLogLevel = androidLogLevel;
+    }
+
+    public int getAndroidLogLevel() {
+        return androidLogLevel;
+    }
+}
diff --git a/mobile/android/fenix/app/src/main/java/com/adjust/sdk/OnAttributionChangedListener.java b/mobile/android/fenix/app/src/main/java/com/adjust/sdk/OnAttributionChangedListener.java
new file mode 100644
index 000000000000..7efa1c6804c6
--- /dev/null
+++ b/mobile/android/fenix/app/src/main/java/com/adjust/sdk/OnAttributionChangedListener.java
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2012-2017 adjust GmbH,
+ * http://www.adjust.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining
+ * a copy of this software and associated documentation files (the
+ * "Software"), to deal in the Software without restriction, including
+ * without limitation the rights to use, copy, modify, merge, publish,
+ * distribute, sublicense, and/or sell copies of the Software, and to
+ * permit persons to whom the Software is furnished to do so, subject to
+ * the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be
+ * included in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+ * LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+ * WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package com.adjust.sdk;
+
+public interface OnAttributionChangedListener {
+    void onAttributionChanged(AdjustAttribution attribution);
+}
diff --git a/mobile/android/fenix/app/src/main/java/com/google/firebase/messaging/FirebaseMessagingService.java b/mobile/android/fenix/app/src/main/java/com/google/firebase/messaging/FirebaseMessagingService.java
new file mode 100644
index 000000000000..4d5fd8153d21
--- /dev/null
+++ b/mobile/android/fenix/app/src/main/java/com/google/firebase/messaging/FirebaseMessagingService.java
@@ -0,0 +1,42 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package com.google.firebase.messaging;
+
+import android.app.Service;
+import android.content.Intent;
+import android.os.Binder;
+import android.os.IBinder;
+
+public class FirebaseMessagingService extends Service {
+
+    private final IBinder mBinder = new Binder();
+
+    public void onMessageReceived(RemoteMessage message) {
+    }
+
+    public void onMessageSent(String msgId) {
+    }
+
+    public void onNewToken(String token) {
+    }
+
+    public void onSendError(String msgId, Exception exception) {
+    }
+
+    @Override
+    public IBinder onBind(Intent intent) {
+        return mBinder;
+    }
+
+}
diff --git a/mobile/android/fenix/app/src/main/java/com/google/firebase/messaging/RemoteMessage.java b/mobile/android/fenix/app/src/main/java/com/google/firebase/messaging/RemoteMessage.java
new file mode 100644
index 000000000000..9ad59a31e43c
--- /dev/null
+++ b/mobile/android/fenix/app/src/main/java/com/google/firebase/messaging/RemoteMessage.java
@@ -0,0 +1,33 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package com.google.firebase.messaging;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+import java.util.Map;
+
+public class RemoteMessage implements Parcelable {
+
+    public int describeContents() {
+        return 0;
+    }
+
+    public void writeToParcel(Parcel out, int flags) {
+    }
+
+    public Map<String, String> getData() {
+        return null;
+    }
+
+}
diff --git a/mobile/android/fenix/app/src/main/java/mozilla/components/lib/push/firebase/AbstractFirebasePushService.kt b/mobile/android/fenix/app/src/main/java/mozilla/components/lib/push/firebase/AbstractFirebasePushService.kt
new file mode 100644
index 000000000000..b50a6f03a218
--- /dev/null
+++ b/mobile/android/fenix/app/src/main/java/mozilla/components/lib/push/firebase/AbstractFirebasePushService.kt
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.lib.push.firebase
+
+import android.content.Context
+import com.google.firebase.messaging.FirebaseMessagingService
+import com.google.firebase.messaging.RemoteMessage
+import mozilla.components.concept.push.PushService
+
+abstract class AbstractFirebasePushService() : FirebaseMessagingService(), PushService {
+
+    override fun start(context: Context) {
+    }
+
+    override fun onNewToken(newToken: String) {
+    }
+
+    override fun onMessageReceived(remoteMessage: RemoteMessage?) {
+    }
+
+    final override fun stop() {
+    }
+
+    override fun deleteToken() {
+    }
+
+    override fun isServiceAvailable(context: Context): Boolean {
+        return false
+    }
+}
diff --git a/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt b/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt
index 67fff191e40c..fd7466e5d4fe 100644
--- a/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt
+++ b/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt
@@ -1211,19 +1211,16 @@ abstract class BaseBrowserFragment :
             view = view,
         )
 
-        // This component feature only works on Fenix when built on Mozilla infrastructure.
-        if (BuildConfig.MOZILLA_OFFICIAL) {
-            webAuthnFeature.set(
-                feature = WebAuthnFeature(
-                    engine = requireComponents.core.engine,
-                    activity = requireActivity(),
-                    exitFullScreen = requireComponents.useCases.sessionUseCases.exitFullscreen::invoke,
-                    currentTab = { store.state.selectedTabId },
-                ),
-                owner = this,
-                view = view,
-            )
-        }
+        webAuthnFeature.set(
+            feature = WebAuthnFeature(
+                engine = requireComponents.core.engine,
+                activity = requireActivity(),
+                exitFullScreen = requireComponents.useCases.sessionUseCases.exitFullscreen::invoke,
+                currentTab = { store.state.selectedTabId },
+            ),
+            owner = this,
+            view = view,
+        )
 
         screenOrientationFeature.set(
             feature = ScreenOrientationFeature(
diff --git a/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/components/Components.kt b/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/components/Components.kt
index 5c75c1462570..d7119f818e64 100644
--- a/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/components/Components.kt
+++ b/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/components/Components.kt
@@ -10,7 +10,6 @@ import androidx.compose.runtime.Composable
 import androidx.compose.runtime.ReadOnlyComposable
 import androidx.compose.ui.platform.LocalContext
 import androidx.core.app.NotificationManagerCompat
-import com.google.android.play.core.review.ReviewManagerFactory
 import mozilla.components.feature.addons.AddonManager
 import mozilla.components.feature.addons.amo.AMOAddonsProvider
 import mozilla.components.feature.addons.migration.DefaultSupportedAddonsChecker
@@ -215,7 +214,6 @@ class Components(private val context: Context) {
 
     val playStoreReviewPromptController by lazyMonitored {
         PlayStoreReviewPromptController(
-            manager = ReviewManagerFactory.create(context),
             numberOfAppLaunches = { settings.numberOfAppLaunches },
         )
     }
diff --git a/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/components/Core.kt b/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/components/Core.kt
index 9d690c06bc74..5b3aa7326e75 100644
--- a/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/components/Core.kt
+++ b/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/components/Core.kt
@@ -620,8 +620,15 @@ class Core(
             if (!context.settings().defaultTopSitesAdded) {
                 defaultTopSites.add(
                     Pair(
-                        context.getString(R.string.default_top_site_google),
-                        SupportUtils.GOOGLE_URL,
+                        context.getString(R.string.default_top_site_fdroid),
+                        SupportUtils.FDROID_URL,
+                    ),
+                )
+
+                defaultTopSites.add(
+                    Pair(
+                        context.getString(R.string.default_top_site_eff),
+                        SupportUtils.EFF_URL,
                     ),
                 )
 
diff --git a/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/components/PlayStoreReviewPromptController.kt b/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/components/PlayStoreReviewPromptController.kt
index 833ff05c35d4..84a7584e13c4 100644
--- a/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/components/PlayStoreReviewPromptController.kt
+++ b/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/components/PlayStoreReviewPromptController.kt
@@ -9,9 +9,6 @@ import android.content.ActivityNotFoundException
 import android.content.Intent
 import androidx.annotation.VisibleForTesting
 import androidx.core.net.toUri
-import com.google.android.play.core.review.ReviewException
-import com.google.android.play.core.review.ReviewInfo
-import com.google.android.play.core.review.ReviewManager
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.withContext
 import mozilla.components.support.base.log.logger.Logger
@@ -29,7 +26,6 @@ val logger = Logger("PlayStoreReviewPromptController")
  * Wraps the Play Store In-App Review API.
  */
 class PlayStoreReviewPromptController(
-    private val manager: ReviewManager,
     private val numberOfAppLaunches: () -> Int,
 ) {
 
@@ -37,60 +33,12 @@ class PlayStoreReviewPromptController(
      * Launch the in-app review flow, unless we've hit the quota.
      */
     suspend fun tryPromptReview(activity: Activity) {
-        logger.info("tryPromptReview in progress...")
-        val reviewInfoFlow = withContext(Dispatchers.IO) { manager.requestReviewFlow() }
-
-        reviewInfoFlow.addOnCompleteListener {
-            val resultString = if (it.isSuccessful) {
-                logger.info("Review flow launched.")
-                // Launch the in-app flow.
-                manager.launchReviewFlow(activity, it.result)
-
-                it.result.toString()
-            } else {
-                // Launch the Play store flow.
-                val reviewErrorCode =
-                    (it.exception as? ReviewException)?.errorCode ?: ERROR_CODE_UNEXPECTED
-                logger.warn("Failed to launch in-app review flow due to: $reviewErrorCode .")
-
-                tryLaunchPlayStoreReview(activity)
-
-                "reviewErrorCode=$reviewErrorCode"
-            }
-
-            recordReviewPromptEvent(
-                reviewInfoAsString = resultString,
-                numberOfAppLaunches = numberOfAppLaunches(),
-                now = Date(),
-            )
-        }
-
-        logger.info("tryPromptReview completed.")
     }
 
     /**
      * Try to launch the play store review flow.
      */
     fun tryLaunchPlayStoreReview(activity: Activity) {
-        logger.info("tryLaunchPlayStoreReview in progress...")
-
-        try {
-            logger.info("Navigating to Play store listing.")
-            activity.startActivity(
-                Intent(Intent.ACTION_VIEW, SupportUtils.RATE_APP_URL.toUri()),
-            )
-        } catch (e: ActivityNotFoundException) {
-            // Device without the play store installed.
-            // Opening the play store website.
-            (activity as HomeActivity).openToBrowserAndLoad(
-                searchTermOrURL = SupportUtils.FENIX_PLAY_STORE_URL,
-                newTab = true,
-                from = BrowserDirection.FromGlobal,
-            )
-            logger.warn("Failed to launch play store review flow due to: $e .")
-        }
-
-        logger.info("tryLaunchPlayStoreReview completed.")
     }
 
     companion object {
diff --git a/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/components/metrics/InstallReferrerMetricsService.kt b/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/components/metrics/InstallReferrerMetricsService.kt
index 233855f4c25b..2af8f7844428 100644
--- a/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/components/metrics/InstallReferrerMetricsService.kt
+++ b/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/components/metrics/InstallReferrerMetricsService.kt
@@ -6,8 +6,6 @@ package org.mozilla.fenix.components.metrics
 
 import android.content.Context
 import android.os.RemoteException
-import com.android.installreferrer.api.InstallReferrerClient
-import com.android.installreferrer.api.InstallReferrerStateListener
 import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.launch
@@ -15,9 +13,6 @@ import mozilla.components.support.base.log.logger.Logger
 import org.json.JSONException
 import org.json.JSONObject
 import org.mozilla.fenix.FeatureFlags
-import org.mozilla.fenix.GleanMetrics.MetaAttribution
-import org.mozilla.fenix.GleanMetrics.Pings
-import org.mozilla.fenix.GleanMetrics.PlayStoreAttribution
 import org.mozilla.fenix.ext.settings
 import org.mozilla.fenix.utils.Settings
 import java.io.UnsupportedEncodingException
@@ -33,86 +28,10 @@ class InstallReferrerMetricsService(private val context: Context) : MetricsServi
     private val logger = Logger("InstallReferrerMetricsService")
     override val type = MetricServiceType.Data
 
-    private var referrerClient: InstallReferrerClient? = null
-
     override fun start() {
-        if (context.settings().utmParamsKnown) {
-            return
-        }
-
-        val timerId = PlayStoreAttribution.attributionTime.start()
-        val client = InstallReferrerClient.newBuilder(context).build()
-        referrerClient = client
-
-        client.startConnection(
-            object : InstallReferrerStateListener {
-                override fun onInstallReferrerSetupFinished(responseCode: Int) {
-                    PlayStoreAttribution.attributionTime.stopAndAccumulate(timerId)
-                    val firstSession = FirstSessionPing(context)
-                    PlayStoreAttribution.responseCode.set(responseCode.toString())
-                    when (responseCode) {
-                        InstallReferrerClient.InstallReferrerResponse.OK -> {
-                            // Connection established.
-                            val installReferrerResponse = try {
-                                client.installReferrer.installReferrer
-                            } catch (e: RemoteException) {
-                                // We can't do anything about this.
-                                logger.error("Failed to retrieve install referrer response", e)
-                                null
-                            } catch (e: SecurityException) {
-                                logger.error("Failed to retrieve install referrer response", e)
-                                null
-                            }
-
-                            if (installReferrerResponse.isNullOrBlank()) {
-                                return
-                            }
-
-                            PlayStoreAttribution.installReferrerResponse.set(installReferrerResponse)
-
-                            val utmParams = UTMParams.parseUTMParameters(installReferrerResponse)
-                            if (FeatureFlags.META_ATTRIBUTION_ENABLED) {
-                                MetaParams.extractMetaAttribution(utmParams.content)
-                                    ?.recordMetaAttribution()
-                            }
-
-                            utmParams.recordInstallReferrer(context.settings())
-                            context.settings().utmParamsKnown = true
-
-                            firstSession.checkAndSend()
-                            triggerPing()
-                        }
-
-                        InstallReferrerClient.InstallReferrerResponse.FEATURE_NOT_SUPPORTED,
-                        InstallReferrerClient.InstallReferrerResponse.DEVELOPER_ERROR,
-                        InstallReferrerClient.InstallReferrerResponse.PERMISSION_ERROR,
-                        -> {
-                            // unrecoverable errors, but we still want to send the first-session ping.
-                            context.settings().utmParamsKnown = true
-                            firstSession.checkAndSend()
-                            triggerPing()
-                        }
-
-                        InstallReferrerClient.InstallReferrerResponse.SERVICE_UNAVAILABLE -> {
-                            // Connection couldn't be established.
-                        }
-                    }
-                    // End the connection, and null out the client.
-                    stop()
-                }
-
-                override fun onInstallReferrerServiceDisconnected() {
-                    // Try to restart the connection on the next request to
-                    // Google Play by calling the startConnection() method.
-                    referrerClient = null
-                }
-            },
-        )
     }
 
     override fun stop() {
-        referrerClient?.endConnection()
-        referrerClient = null
     }
 
     override fun track(event: Event) = Unit
@@ -120,9 +39,6 @@ class InstallReferrerMetricsService(private val context: Context) : MetricsServi
     override fun shouldTrack(event: Event): Boolean = false
 
     private fun triggerPing() {
-        CoroutineScope(Dispatchers.IO).launch {
-            Pings.playStoreAttribution.submit()
-        }
     }
 }
 
@@ -237,12 +153,6 @@ data class UTMParams(
             return
         }
         intoSettings(settings)
-
-        PlayStoreAttribution.source.set(source)
-        PlayStoreAttribution.medium.set(medium)
-        PlayStoreAttribution.campaign.set(campaign)
-        PlayStoreAttribution.content.set(content)
-        PlayStoreAttribution.term.set(term)
     }
 }
 
@@ -337,9 +247,5 @@ data class MetaParams(
      * record META attribution params to telemetry
      */
     fun recordMetaAttribution() {
-        MetaAttribution.app.set(app)
-        MetaAttribution.t.set(t)
-        MetaAttribution.data.set(data)
-        MetaAttribution.nonce.set(nonce)
     }
 }
diff --git a/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/components/metrics/MarketingAttributionService.kt b/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/components/metrics/MarketingAttributionService.kt
index f1d549f34da2..e17ece068589 100644
--- a/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/components/metrics/MarketingAttributionService.kt
+++ b/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/components/metrics/MarketingAttributionService.kt
@@ -7,8 +7,6 @@ package org.mozilla.fenix.components.metrics
 import android.content.Context
 import android.os.RemoteException
 import androidx.annotation.VisibleForTesting
-import com.android.installreferrer.api.InstallReferrerClient
-import com.android.installreferrer.api.InstallReferrerStateListener
 import mozilla.components.support.base.log.logger.Logger
 import org.mozilla.fenix.ext.components
 import org.mozilla.fenix.ext.settings
@@ -24,74 +22,17 @@ const val ADJUST_REFTAG_PREFIX = "adjust_reftag="
  */
 class MarketingAttributionService(private val context: Context) {
     private val logger = Logger("MarketingAttributionService")
-    private var referrerClient: InstallReferrerClient? = null
 
     /**
      * Starts the connection with the install referrer and handle the response.
      */
     fun start() {
-        val client = InstallReferrerClient.newBuilder(context).build()
-        referrerClient = client
-
-        client.startConnection(
-            object : InstallReferrerStateListener {
-                override fun onInstallReferrerSetupFinished(responseCode: Int) {
-                    when (responseCode) {
-                        InstallReferrerClient.InstallReferrerResponse.OK -> {
-                            // Connection established.
-                            val installReferrerResponse = try {
-                                client.installReferrer.installReferrer
-                            } catch (e: RemoteException) {
-                                // We can't do anything about this.
-                                logger.error("Failed to retrieve install referrer response", e)
-                                null
-                            } catch (e: SecurityException) {
-                                // https://issuetracker.google.com/issues/72926755
-                                logger.error("Failed to retrieve install referrer response", e)
-                                null
-                            }
-
-                            if (!installReferrerResponse.isNullOrBlank()) {
-                                response = installReferrerResponse
-                                val utmParams = UTMParams.parseUTMParameters(installReferrerResponse)
-
-                                context.components.distributionIdManager
-                                    .updateDistributionIdFromUtmParams(utmParams)
-                            }
-
-                            context.settings().shouldShowMarketingOnboarding =
-                                shouldShowMarketingOnboarding(installReferrerResponse)
-
-                            return
-                        }
-
-                        InstallReferrerClient.InstallReferrerResponse.FEATURE_NOT_SUPPORTED,
-                        InstallReferrerClient.InstallReferrerResponse.DEVELOPER_ERROR,
-                        InstallReferrerClient.InstallReferrerResponse.PERMISSION_ERROR,
-                        InstallReferrerClient.InstallReferrerResponse.SERVICE_UNAVAILABLE,
-                        -> {
-                            context.settings().shouldShowMarketingOnboarding = false
-                            return
-                        }
-                    }
-
-                    // End the connection, and null out the client.
-                    stop()
-                }
-
-                override fun onInstallReferrerServiceDisconnected() {
-                    referrerClient = null
-                }
-            },
-        )
     }
 
     /**
      * Stops the connection with the install referrer.
      */
     fun stop() {
-        referrerClient?.endConnection()
-        referrerClient = null
     }
 
     /**
diff --git a/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/home/pocket/ContentRecommendationsFeatureHelper.kt b/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/home/pocket/ContentRecommendationsFeatureHelper.kt
index d211d3625dbf..0458e8704d5c 100644
--- a/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/home/pocket/ContentRecommendationsFeatureHelper.kt
+++ b/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/home/pocket/ContentRecommendationsFeatureHelper.kt
@@ -16,22 +16,7 @@ object ContentRecommendationsFeatureHelper {
     /**
      * List of supported content recommendations locales.
      */
-    val CONTENT_RECOMMENDATIONS_SUPPORTED_LOCALE = listOf(
-        "fr",
-        "fr-FR",
-        "es",
-        "es-ES",
-        "it",
-        "it-IT",
-        "en",
-        "en-CA",
-        "en-GB",
-        "en-US",
-        "de",
-        "de-DE",
-        "de-AT",
-        "de-CH",
-    )
+    val CONTENT_RECOMMENDATIONS_SUPPORTED_LOCALE = listOf<String>()
 
     /**
      * Show Pocket sponsored stories in between Pocket recommended stories on home.
diff --git a/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/onboarding/OnboardingFragment.kt b/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/onboarding/OnboardingFragment.kt
index 8d46a483fcff..cef6afd4d01e 100644
--- a/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/onboarding/OnboardingFragment.kt
+++ b/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/onboarding/OnboardingFragment.kt
@@ -455,8 +455,7 @@ class OnboardingFragment : Fragment() {
         }
 
         if (!settings.isTelemetryEnabled) {
-            Pings.onboardingOptOut.setEnabled(true)
-            Pings.onboardingOptOut.submit()
+            Pings.onboardingOptOut.setEnabled(false)
         }
 
         startMetricsIfEnabled(
diff --git a/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/settings/SecretSettingsFragment.kt b/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/settings/SecretSettingsFragment.kt
index 40212e28e9e0..6f8455fa6f8a 100644
--- a/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/settings/SecretSettingsFragment.kt
+++ b/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/settings/SecretSettingsFragment.kt
@@ -297,28 +297,6 @@ class SecretSettingsFragment : PreferenceFragmentCompat() {
             isVisible = Config.channel.isNightlyOrDebug
         }
 
-        requirePreference<SwitchPreference>(R.string.pref_key_remote_server_prod).apply {
-            isVisible = true
-            isChecked = context.settings().useProductionRemoteSettingsServer
-            onPreferenceChangeListener = object : SharedPreferenceUpdater() {
-                override fun onPreferenceChange(preference: Preference, newValue: Any?): Boolean {
-                    val service =
-                        context.components.remoteSettingsService.value.remoteSettingsService
-                    service.updateConfig(
-                        RemoteSettingsServerConfig(
-                            server = if (newValue as? Boolean == true) {
-                                RemoteSettingsServer.Prod.into()
-                            } else {
-                                RemoteSettingsServer.Stage.into()
-                            },
-                        ).into(),
-                    )
-                    service.sync()
-                    return super.onPreferenceChange(preference, newValue)
-                }
-            }
-        }
-
         requirePreference<SwitchPreference>(R.string.pref_key_use_remote_search_configuration).apply {
             isVisible = true
             isChecked = context.settings().useRemoteSearchConfiguration
diff --git a/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/settings/SupportUtils.kt b/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/settings/SupportUtils.kt
index 6ca1dca238cb..169849c0d1aa 100644
--- a/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/settings/SupportUtils.kt
+++ b/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/settings/SupportUtils.kt
@@ -30,6 +30,8 @@ object SupportUtils {
     const val WHATS_NEW_URL = "https://www.mozilla.org/firefox/android/notes"
     const val FXACCOUNT_SUMO_URL = "https://support.mozilla.org/kb/access-mozilla-services-firefox-account"
     const val ANDROID_SUPPORT_SUMO_URL = "mzl.la/AndroidSupport"
+    const val FDROID_URL = "https://f-droid.org/"
+    const val EFF_URL = "https://www.eff.org/"
 
     // This is locale-less on purpose so that the content negotiation happens on the AMO side because the current
     // user language might not be supported by AMO and/or the language might not be exactly what AMO is expecting
diff --git a/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/utils/Settings.kt b/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/utils/Settings.kt
index 6ef1bc0ace33..c6fbe34446e4 100644
--- a/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/utils/Settings.kt
+++ b/mobile/android/fenix/app/src/main/java/org/mozilla/fenix/utils/Settings.kt
@@ -166,8 +166,7 @@ class Settings(private val appContext: Context) : PreferencesHolder {
     private val logger = Logger("Settings")
 
     @VisibleForTesting
-    internal val isCrashReportEnabledInBuild: Boolean =
-        BuildConfig.CRASH_REPORTING && Config.channel.isReleased
+    internal val isCrashReportEnabledInBuild: Boolean = false
 
     override val preferences: SharedPreferences =
         appContext.getSharedPreferences(FENIX_PREFERENCES, MODE_PRIVATE)
@@ -193,20 +192,12 @@ class Settings(private val appContext: Context) : PreferencesHolder {
     /**
      * Indicates if the stories homescreen section should be shown.
      */
-    var showPocketRecommendationsFeature by lazyFeatureFlagPreference(
-        appContext.getPreferenceKey(R.string.pref_key_pocket_homescreen_recommendations),
-        featureFlag = ContentRecommendationsFeatureHelper.isContentRecommendationsFeatureEnabled(appContext),
-        default = { homescreenSections[HomeScreenSection.POCKET] == true },
-    )
+    var showPocketRecommendationsFeature = false
 
     /**
      * Indicates if the Pocket recommendations homescreen section should also show sponsored stories.
      */
-    val showPocketSponsoredStories by lazyFeatureFlagPreference(
-        key = appContext.getPreferenceKey(R.string.pref_key_pocket_sponsored_stories),
-        default = { homescreenSections[HomeScreenSection.POCKET_SPONSORED_STORIES] == true },
-        featureFlag = ContentRecommendationsFeatureHelper.isPocketSponsoredStoriesFeatureEnabled(appContext),
-    )
+    val showPocketSponsoredStories = false
 
     /**
      * Indicates whether or not the "Recently Visited" section should be shown on the home screen.
@@ -503,12 +494,7 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         default = true,
     )
 
-    val isCrashReportingEnabled: Boolean
-        get() = isCrashReportEnabledInBuild &&
-            preferences.getBoolean(
-                appContext.getPreferenceKey(R.string.pref_key_crash_reporter),
-                true,
-            )
+    val isCrashReportingEnabled: Boolean = false
 
     var crashReportChoice by stringPreference(
         appContext.getPreferenceKey(R.string.pref_key_crash_reporting_choice),
@@ -520,20 +506,11 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         default = false,
     )
 
-    var isTelemetryEnabled by booleanPreference(
-        appContext.getPreferenceKey(R.string.pref_key_telemetry),
-        default = true,
-    )
+    var isTelemetryEnabled = false
 
-    var isMarketingTelemetryEnabled by booleanPreference(
-        appContext.getPreferenceKey(R.string.pref_key_marketing_telemetry),
-        default = false,
-    )
+    var isMarketingTelemetryEnabled = false
 
-    var hasMadeMarketingTelemetrySelection by booleanPreference(
-        appContext.getPreferenceKey(R.string.pref_key_marketing_telemetry_selection_made),
-        default = false,
-    )
+    var hasMadeMarketingTelemetrySelection = false
 
     var hasAcceptedTermsOfService by booleanPreference(
         appContext.getPreferenceKey(R.string.pref_key_terms_accepted),
@@ -611,16 +588,9 @@ class Settings(private val appContext: Context) : PreferencesHolder {
      * sure that users who upgrade and had telemetry disabled don't start sending the
      * daily usage ping telemetry.
      */
-    var isDailyUsagePingEnabled by booleanPreference(
-        appContext.getPreferenceKey(R.string.pref_key_daily_usage_ping),
-        default = isTelemetryEnabled,
-        persistDefaultIfNotExists = true,
-    )
+    var isDailyUsagePingEnabled = false
 
-    var isExperimentationEnabled by booleanPreference(
-        appContext.getPreferenceKey(R.string.pref_key_experimentation_v2),
-        default = isTelemetryEnabled,
-    )
+    var isExperimentationEnabled = false
 
     /**
      * This lets us know if the user has disabled experimentation manually so that we know
@@ -2048,10 +2018,7 @@ class Settings(private val appContext: Context) : PreferencesHolder {
     /**
      * Indicates if the Contile functionality should be visible.
      */
-    var showContileFeature by booleanPreference(
-        key = appContext.getPreferenceKey(R.string.pref_key_enable_contile),
-        default = true,
-    )
+    var showContileFeature = false
 
     /**
      * Indicates if the Unified Search feature should be visible.
diff --git a/mobile/android/fenix/app/src/main/res/values/static_strings_extra.xml b/mobile/android/fenix/app/src/main/res/values/static_strings_extra.xml
new file mode 100644
index 000000000000..101f4e0d0a90
--- /dev/null
+++ b/mobile/android/fenix/app/src/main/res/values/static_strings_extra.xml
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<resources>
+    <string name="default_top_site_fdroid" translatable="false">F-Droid</string>
+    <string name="default_top_site_eff" translatable="false">EFF</string>
+</resources>
diff --git a/mobile/android/fenix/app/src/main/res/xml/home_preferences.xml b/mobile/android/fenix/app/src/main/res/xml/home_preferences.xml
index 4fd1f8945ae2..9d6da836cf30 100644
--- a/mobile/android/fenix/app/src/main/res/xml/home_preferences.xml
+++ b/mobile/android/fenix/app/src/main/res/xml/home_preferences.xml
@@ -13,7 +13,8 @@
         android:dependency="@string/pref_key_show_top_sites"
         android:layout="@layout/checkbox_left_sub_preference"
         android:key="@string/pref_key_enable_contile"
-        android:title="@string/customize_toggle_contile" />
+        android:title="@string/customize_toggle_contile"
+        app:isPreferenceVisible="false" />
 
     <androidx.preference.SwitchPreference
         android:key="@string/pref_key_recent_tabs"
@@ -29,13 +30,15 @@
 
     <androidx.preference.SwitchPreference
         android:key="@string/pref_key_pocket_homescreen_recommendations"
-        android:title="@string/customize_toggle_pocket_3" />
+        android:title="@string/customize_toggle_pocket_3"
+        app:isPreferenceVisible="false" />
 
     <androidx.preference.CheckBoxPreference
         android:dependency="@string/pref_key_pocket_homescreen_recommendations"
         android:layout="@layout/checkbox_left_sub_preference"
         android:key="@string/pref_key_pocket_sponsored_stories"
-        android:title="@string/customize_toggle_pocket_sponsored" />
+        android:title="@string/customize_toggle_pocket_sponsored"
+        app:isPreferenceVisible="false" />
 
     <androidx.preference.Preference
         android:key="@string/pref_key_wallpapers"
diff --git a/mobile/android/fenix/app/src/main/res/xml/preferences.xml b/mobile/android/fenix/app/src/main/res/xml/preferences.xml
index 159944ab7d38..d3480d93a5b2 100644
--- a/mobile/android/fenix/app/src/main/res/xml/preferences.xml
+++ b/mobile/android/fenix/app/src/main/res/xml/preferences.xml
@@ -141,11 +141,6 @@
             app:iconSpaceReserved="false"
             android:title="@string/preferences_notifications" />
 
-        <androidx.preference.Preference
-            android:key="@string/pref_key_data_choices"
-            app:iconSpaceReserved="false"
-            android:title="@string/preferences_data_collection" />
-
     </androidx.preference.PreferenceCategory>
 
     <PreferenceCategory
@@ -206,11 +201,6 @@
         android:title="@string/preferences_category_about"
         app:iconSpaceReserved="false"
         android:layout="@layout/preference_category_no_icon_style">
-        <androidx.preference.Preference
-            android:key="@string/pref_key_rate"
-            app:iconSpaceReserved="false"
-            android:title="@string/preferences_rate" />
-
         <androidx.preference.Preference
             android:key="@string/pref_key_about"
             app:iconSpaceReserved="false"
diff --git a/mobile/android/fenix/app/src/main/res/xml/secret_settings_preferences.xml b/mobile/android/fenix/app/src/main/res/xml/secret_settings_preferences.xml
index e8059b6aea0b..a0747129882e 100644
--- a/mobile/android/fenix/app/src/main/res/xml/secret_settings_preferences.xml
+++ b/mobile/android/fenix/app/src/main/res/xml/secret_settings_preferences.xml
@@ -138,6 +138,7 @@
     <SwitchPreference
         android:key="@string/pref_key_remote_server_prod"
         app:iconSpaceReserved="false"
+        app:isPreferenceVisible="false"
         android:title="@string/preferences_debug_settings_remote_settings_server"
         />
     <SwitchPreference
diff --git a/mobile/android/fenix/app/src/main/res/xml/site_permissions_details_exceptions_preferences.xml b/mobile/android/fenix/app/src/main/res/xml/site_permissions_details_exceptions_preferences.xml
index dfe2d28572f2..54a99d9e1dc5 100644
--- a/mobile/android/fenix/app/src/main/res/xml/site_permissions_details_exceptions_preferences.xml
+++ b/mobile/android/fenix/app/src/main/res/xml/site_permissions_details_exceptions_preferences.xml
@@ -44,7 +44,8 @@
         android:icon="@drawable/ic_link"
         android:key="@string/pref_key_browser_feature_media_key_system_access"
         android:title="@string/preference_phone_feature_media_key_system_access"
-        android:summary="@string/preference_option_phone_feature_ask_to_allow"/>
+        android:summary="@string/preference_option_phone_feature_ask_to_allow"
+        app:isPreferenceVisible="false"/>
 
     <androidx.preference.Preference
         android:icon="@drawable/ic_autoplay"
diff --git a/mobile/android/fenix/app/src/main/res/xml/site_permissions_preferences.xml b/mobile/android/fenix/app/src/main/res/xml/site_permissions_preferences.xml
index d224eedd7c87..2d1fc926bb34 100644
--- a/mobile/android/fenix/app/src/main/res/xml/site_permissions_preferences.xml
+++ b/mobile/android/fenix/app/src/main/res/xml/site_permissions_preferences.xml
@@ -73,6 +73,7 @@
             android:key="@string/pref_key_browser_feature_media_key_system_access"
             android:title="@string/preference_phone_feature_media_key_system_access"
             android:summary="@string/preference_option_phone_feature_ask_to_allow"
+            app:isPreferenceVisible="false"
             app:allowDividerBelow="true"/>
 
         <androidx.preference.Preference
diff --git a/mobile/android/focus-android/app/build.gradle b/mobile/android/focus-android/app/build.gradle
index ad61a6834166..6c9ddc8fa105 100644
--- a/mobile/android/focus-android/app/build.gradle
+++ b/mobile/android/focus-android/app/build.gradle
@@ -331,8 +331,6 @@ dependencies {
     implementation libs.google.material
     implementation libs.kotlinx.coroutines
     implementation libs.mozilla.glean
-    implementation libs.play.review
-    implementation libs.play.review.ktx
     implementation libs.sentry
 
     debugImplementation libs.leakcanary
