diff -r c3bba5162c98 build.gradle
--- a/build.gradle	Mon Apr 21 16:33:21 2025 +0000
+++ b/build.gradle	Wed Apr 23 09:00:07 2025 +0000
@@ -4,14 +4,9 @@
 
 buildscript {
     repositories {
-        gradle.mozconfig.substs.GRADLE_MAVEN_REPOSITORIES.each { repository ->
-            maven {
-                url = repository
-                if (gradle.mozconfig.substs.ALLOW_INSECURE_GRADLE_REPOSITORIES) {
-                    allowInsecureProtocol = true
-                }
-            }
-        }
+        mavenLocal()
+        maven { url "https://plugins.gradle.org/m2/" }
+        google()
     }
 
     dependencies {
@@ -128,14 +123,9 @@
     }
 
     repositories {
-        gradle.mozconfig.substs.GRADLE_MAVEN_REPOSITORIES.each { repository ->
-            maven {
-                url = repository
-                if (gradle.mozconfig.substs.ALLOW_INSECURE_GRADLE_REPOSITORIES) {
-                    allowInsecureProtocol = true
-                }
-            }
-        }
+        mavenLocal()
+        maven { url "https://plugins.gradle.org/m2/" }
+        google()
     }
 
     // Use the semanticdb-javac and semanticdb-kotlinc plugins to generate semanticdb files for Searchfox
diff -r c3bba5162c98 mobile/android/android-components/build.gradle
--- a/mobile/android/android-components/build.gradle	Mon Apr 21 16:33:21 2025 +0000
+++ b/mobile/android/android-components/build.gradle	Wed Apr 23 09:00:07 2025 +0000
@@ -9,14 +9,9 @@
 
 buildscript {
     repositories {
-        gradle.mozconfig.substs.GRADLE_MAVEN_REPOSITORIES.each { repository ->
-            maven {
-                url = repository
-                if (gradle.mozconfig.substs.ALLOW_INSECURE_GRADLE_REPOSITORIES) {
-                    allowInsecureProtocol = true
-                }
-            }
-        }
+        mavenLocal()
+        maven { url "https://plugins.gradle.org/m2/" }
+        google()
     }
 
     dependencies {
@@ -33,18 +28,10 @@
 
 allprojects {
     repositories {
-        gradle.mozconfig.substs.GRADLE_MAVEN_REPOSITORIES.each { repository ->
-            maven {
-                url = repository
-                if (gradle.mozconfig.substs.ALLOW_INSECURE_GRADLE_REPOSITORIES) {
-                    allowInsecureProtocol = true
-                }
-            }
-        }
+        mavenLocal()
+        maven { url "https://plugins.gradle.org/m2/" }
+        google()
 
-        maven {
-            url = "${gradle.mozconfig.topobjdir}/gradle/maven"
-        }
     }
 
     // Enable Kotlin warnings as errors for all modules
diff -r c3bba5162c98 mobile/android/android-components/components/browser/engine-gecko/build.gradle
--- a/mobile/android/android-components/components/browser/engine-gecko/build.gradle	Mon Apr 21 16:33:21 2025 +0000
+++ b/mobile/android/android-components/components/browser/engine-gecko/build.gradle	Wed Apr 23 09:00:07 2025 +0000
@@ -4,14 +4,9 @@
 
 buildscript {
     repositories {
-        gradle.mozconfig.substs.GRADLE_MAVEN_REPOSITORIES.each { repository ->
-            maven {
-                url = repository
-                if (gradle.mozconfig.substs.ALLOW_INSECURE_GRADLE_REPOSITORIES) {
-                    allowInsecureProtocol = true
-                }
-            }
-        }
+        mavenLocal()
+        maven { url "https://plugins.gradle.org/m2/" }
+        google()
     }
 
     dependencies {
diff -r c3bba5162c98 mobile/android/android-components/components/lib/crash/build.gradle
--- a/mobile/android/android-components/components/lib/crash/build.gradle	Mon Apr 21 16:33:21 2025 +0000
+++ b/mobile/android/android-components/components/lib/crash/build.gradle	Wed Apr 23 09:00:07 2025 +0000
@@ -4,14 +4,9 @@
 
 buildscript {
     repositories {
-        gradle.mozconfig.substs.GRADLE_MAVEN_REPOSITORIES.each { repository ->
-            maven {
-                url = repository
-                if (gradle.mozconfig.substs.ALLOW_INSECURE_GRADLE_REPOSITORIES) {
-                    allowInsecureProtocol = true
-                }
-            }
-        }
+        mavenLocal()
+        maven { url "https://plugins.gradle.org/m2/" }
+        google()
     }
 
     dependencies {
diff -r c3bba5162c98 mobile/android/android-components/components/service/mars/build.gradle
--- a/mobile/android/android-components/components/service/mars/build.gradle	Mon Apr 21 16:33:21 2025 +0000
+++ b/mobile/android/android-components/components/service/mars/build.gradle	Wed Apr 23 09:00:07 2025 +0000
@@ -4,14 +4,9 @@
 
 buildscript {
     repositories {
-        gradle.mozconfig.substs.GRADLE_MAVEN_REPOSITORIES.each { repository ->
-            maven {
-                url = repository
-                if (gradle.mozconfig.substs.ALLOW_INSECURE_GRADLE_REPOSITORIES) {
-                    allowInsecureProtocol = true
-                }
-            }
-        }
+        mavenLocal()
+        maven { url "https://plugins.gradle.org/m2/" }
+        google()
     }
 
     dependencies {
diff -r c3bba5162c98 mobile/android/android-components/components/service/nimbus/build.gradle
--- a/mobile/android/android-components/components/service/nimbus/build.gradle	Mon Apr 21 16:33:21 2025 +0000
+++ b/mobile/android/android-components/components/service/nimbus/build.gradle	Wed Apr 23 09:00:07 2025 +0000
@@ -4,14 +4,9 @@
 
 buildscript {
     repositories {
-        gradle.mozconfig.substs.GRADLE_MAVEN_REPOSITORIES.each { repository ->
-            maven {
-                url = repository
-                if (gradle.mozconfig.substs.ALLOW_INSECURE_GRADLE_REPOSITORIES) {
-                    allowInsecureProtocol = true
-                }
-            }
-        }
+        mavenLocal()
+        maven { url "https://plugins.gradle.org/m2/" }
+        google()
     }
 
     dependencies {
diff -r c3bba5162c98 mobile/android/android-components/components/service/pocket/build.gradle
--- a/mobile/android/android-components/components/service/pocket/build.gradle	Mon Apr 21 16:33:21 2025 +0000
+++ b/mobile/android/android-components/components/service/pocket/build.gradle	Wed Apr 23 09:00:07 2025 +0000
@@ -4,14 +4,9 @@
 
 buildscript {
     repositories {
-        gradle.mozconfig.substs.GRADLE_MAVEN_REPOSITORIES.each { repository ->
-            maven {
-                url = repository
-                if (gradle.mozconfig.substs.ALLOW_INSECURE_GRADLE_REPOSITORIES) {
-                    allowInsecureProtocol = true
-                }
-            }
-        }
+        mavenLocal()
+        maven { url "https://plugins.gradle.org/m2/" }
+        google()
     }
 
     dependencies {
diff -r c3bba5162c98 mobile/android/android-components/components/support/remotesettings/build.gradle
--- a/mobile/android/android-components/components/support/remotesettings/build.gradle	Mon Apr 21 16:33:21 2025 +0000
+++ b/mobile/android/android-components/components/support/remotesettings/build.gradle	Wed Apr 23 09:00:07 2025 +0000
@@ -3,14 +3,9 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 buildscript {
     repositories {
-        gradle.mozconfig.substs.GRADLE_MAVEN_REPOSITORIES.each { repository ->
-            maven {
-                url = repository
-                if (gradle.mozconfig.substs.ALLOW_INSECURE_GRADLE_REPOSITORIES) {
-                    allowInsecureProtocol = true
-                }
-            }
-        }
+        mavenLocal()
+        maven { url "https://plugins.gradle.org/m2/" }
+        google()
     }
     dependencies {
         classpath  libs.plugin.serialization
diff -r c3bba5162c98 mobile/android/android-components/plugins/config/build.gradle
--- a/mobile/android/android-components/plugins/config/build.gradle	Mon Apr 21 16:33:21 2025 +0000
+++ b/mobile/android/android-components/plugins/config/build.gradle	Wed Apr 23 09:00:07 2025 +0000
@@ -9,14 +9,9 @@
 buildDir = "${gradle.mozconfig.topobjdir}/gradle/build/mobile/android/android-components/plugins/config"
 
 repositories {
-    gradle.mozconfig.substs.GRADLE_MAVEN_REPOSITORIES.each { repository ->
-        maven {
-            url = repository
-            if (gradle.mozconfig.substs.ALLOW_INSECURE_GRADLE_REPOSITORIES) {
-                allowInsecureProtocol = true
-            }
-        }
-    }
+    mavenLocal()
+    maven { url "https://plugins.gradle.org/m2/" }
+    google()
 }
 
 gradlePlugin {
diff -r c3bba5162c98 mobile/android/android-components/plugins/config/settings.gradle
--- a/mobile/android/android-components/plugins/config/settings.gradle	Mon Apr 21 16:33:21 2025 +0000
+++ b/mobile/android/android-components/plugins/config/settings.gradle	Wed Apr 23 09:00:07 2025 +0000
@@ -11,14 +11,9 @@
     }
 
     repositories {
-        gradle.mozconfig.substs.GRADLE_MAVEN_REPOSITORIES.each { repository ->
-            maven {
-                url = repository
-                if (gradle.mozconfig.substs.ALLOW_INSECURE_GRADLE_REPOSITORIES) {
-                    allowInsecureProtocol = true
-                }
-            }
-        }
+        mavenLocal()
+        maven { url "https://plugins.gradle.org/m2/" }
+        google()
     }
 }
 
diff -r c3bba5162c98 mobile/android/android-components/plugins/dependencies/build.gradle
--- a/mobile/android/android-components/plugins/dependencies/build.gradle	Mon Apr 21 16:33:21 2025 +0000
+++ b/mobile/android/android-components/plugins/dependencies/build.gradle	Wed Apr 23 09:00:07 2025 +0000
@@ -9,14 +9,9 @@
 buildDir = "${gradle.mozconfig.topobjdir}/gradle/build/mobile/android/android-components/plugins/dependencies"
 
 repositories {
-        gradle.mozconfig.substs.GRADLE_MAVEN_REPOSITORIES.each { repository ->
-            maven {
-                url = repository
-                if (gradle.mozconfig.substs.ALLOW_INSECURE_GRADLE_REPOSITORIES) {
-                    allowInsecureProtocol = true
-                }
-            }
-        }
+    mavenLocal()
+    maven { url "https://plugins.gradle.org/m2/" }
+    google()
 }
 
 gradlePlugin {
diff -r c3bba5162c98 mobile/android/android-components/plugins/dependencies/settings.gradle
--- a/mobile/android/android-components/plugins/dependencies/settings.gradle	Mon Apr 21 16:33:21 2025 +0000
+++ b/mobile/android/android-components/plugins/dependencies/settings.gradle	Wed Apr 23 09:00:07 2025 +0000
@@ -11,14 +11,9 @@
     }
 
     repositories {
-        gradle.mozconfig.substs.GRADLE_MAVEN_REPOSITORIES.each { repository ->
-            maven {
-                url = repository
-                if (gradle.mozconfig.substs.ALLOW_INSECURE_GRADLE_REPOSITORIES) {
-                    allowInsecureProtocol = true
-                }
-            }
-        }
+        mavenLocal()
+        maven { url "https://plugins.gradle.org/m2/" }
+        google()
     }
 }
 
diff -r c3bba5162c98 mobile/android/android-components/plugins/publicsuffixlist/build.gradle
--- a/mobile/android/android-components/plugins/publicsuffixlist/build.gradle	Mon Apr 21 16:33:21 2025 +0000
+++ b/mobile/android/android-components/plugins/publicsuffixlist/build.gradle	Wed Apr 23 09:00:07 2025 +0000
@@ -9,14 +9,9 @@
 buildDir = "${gradle.mozconfig.topobjdir}/gradle/build/mobile/android/android-components/plugins/publicsuffixlist"
 
 repositories {
-        gradle.mozconfig.substs.GRADLE_MAVEN_REPOSITORIES.each { repository ->
-            maven {
-                url = repository
-                if (gradle.mozconfig.substs.ALLOW_INSECURE_GRADLE_REPOSITORIES) {
-                    allowInsecureProtocol = true
-                }
-            }
-        }
+    mavenLocal()
+    maven { url "https://plugins.gradle.org/m2/" }
+    google()
 }
 
 dependencies {
diff -r c3bba5162c98 mobile/android/android-components/plugins/publicsuffixlist/settings.gradle
--- a/mobile/android/android-components/plugins/publicsuffixlist/settings.gradle	Mon Apr 21 16:33:21 2025 +0000
+++ b/mobile/android/android-components/plugins/publicsuffixlist/settings.gradle	Wed Apr 23 09:00:07 2025 +0000
@@ -12,14 +12,9 @@
     }
 
     repositories {
-        gradle.mozconfig.substs.GRADLE_MAVEN_REPOSITORIES.each { repository ->
-            maven {
-                url = repository
-                if (gradle.mozconfig.substs.ALLOW_INSECURE_GRADLE_REPOSITORIES) {
-                    allowInsecureProtocol = true
-                }
-            }
-        }
+        mavenLocal()
+        maven { url "https://plugins.gradle.org/m2/" }
+        google()
     }
 
     includeBuild("../dependencies")
diff -r c3bba5162c98 mobile/android/android-components/samples/glean/build.gradle
--- a/mobile/android/android-components/samples/glean/build.gradle	Mon Apr 21 16:33:21 2025 +0000
+++ b/mobile/android/android-components/samples/glean/build.gradle	Wed Apr 23 09:00:07 2025 +0000
@@ -4,14 +4,9 @@
 
 buildscript {
     repositories {
-        gradle.mozconfig.substs.GRADLE_MAVEN_REPOSITORIES.each { repository ->
-            maven {
-                url = repository
-                if (gradle.mozconfig.substs.ALLOW_INSECURE_GRADLE_REPOSITORIES) {
-                    allowInsecureProtocol = true
-                }
-            }
-        }
+        mavenLocal()
+        maven { url "https://plugins.gradle.org/m2/" }
+        google()
 
         dependencies {
             classpath "org.mozilla.telemetry:glean-gradle-plugin:${libs.versions.mozilla.glean.get()}"
diff -r c3bba5162c98 mobile/android/android-components/samples/glean/samples-glean-library/build.gradle
--- a/mobile/android/android-components/samples/glean/samples-glean-library/build.gradle	Mon Apr 21 16:33:21 2025 +0000
+++ b/mobile/android/android-components/samples/glean/samples-glean-library/build.gradle	Wed Apr 23 09:00:07 2025 +0000
@@ -4,14 +4,9 @@
 
 buildscript {
     repositories {
-        gradle.mozconfig.substs.GRADLE_MAVEN_REPOSITORIES.each { repository ->
-            maven {
-                url = repository
-                if (gradle.mozconfig.substs.ALLOW_INSECURE_GRADLE_REPOSITORIES) {
-                    allowInsecureProtocol = true
-                }
-            }
-        }
+        mavenLocal()
+        maven { url "https://plugins.gradle.org/m2/" }
+        google()
 
         dependencies {
             classpath "org.mozilla.telemetry:glean-gradle-plugin:${libs.versions.mozilla.glean.get()}"
diff -r c3bba5162c98 mobile/android/android-components/settings.gradle
--- a/mobile/android/android-components/settings.gradle	Mon Apr 21 16:33:21 2025 +0000
+++ b/mobile/android/android-components/settings.gradle	Wed Apr 23 09:00:07 2025 +0000
@@ -10,14 +10,9 @@
     }
 
     repositories {
-        gradle.mozconfig.substs.GRADLE_MAVEN_REPOSITORIES.each { repository ->
-            maven {
-                url = repository
-                if (gradle.mozconfig.substs.ALLOW_INSECURE_GRADLE_REPOSITORIES) {
-                    allowInsecureProtocol = true
-                }
-            }
-        }
+        mavenLocal()
+        maven { url "https://plugins.gradle.org/m2/" }
+        google()
     }
 
     includeBuild("plugins/dependencies")
diff -r c3bba5162c98 mobile/android/fenix/app/build.gradle
--- a/mobile/android/fenix/app/build.gradle	Mon Apr 21 16:33:21 2025 +0000
+++ b/mobile/android/fenix/app/build.gradle	Wed Apr 23 09:00:07 2025 +0000
@@ -3,14 +3,9 @@
 
 buildscript {
     repositories {
-        gradle.mozconfig.substs.GRADLE_MAVEN_REPOSITORIES.each { repository ->
-            maven {
-                url = repository
-                if (gradle.mozconfig.substs.ALLOW_INSECURE_GRADLE_REPOSITORIES) {
-                    allowInsecureProtocol = true
-                }
-            }
-        }
+        mavenLocal()
+        maven { url "https://plugins.gradle.org/m2/" }
+        google()
     }
 
     dependencies {
diff -r c3bba5162c98 mobile/android/fenix/build.gradle
--- a/mobile/android/fenix/build.gradle	Mon Apr 21 16:33:21 2025 +0000
+++ b/mobile/android/fenix/build.gradle	Wed Apr 23 09:00:07 2025 +0000
@@ -10,14 +10,9 @@
 buildscript {
     // This logic is duplicated in the allprojects block: I don't know how to fix that.
     repositories {
-        gradle.mozconfig.substs.GRADLE_MAVEN_REPOSITORIES.each { repository ->
-            maven {
-                url = repository
-                if (gradle.mozconfig.substs.ALLOW_INSECURE_GRADLE_REPOSITORIES) {
-                    allowInsecureProtocol = true
-                }
-            }
-        }
+        mavenLocal()
+        maven { url "https://plugins.gradle.org/m2/" }
+        google()
     }
 
     dependencies {
@@ -45,18 +40,10 @@
 allprojects {
     // This logic is duplicated in the buildscript block: I don't know how to fix that.
     repositories {
-        gradle.mozconfig.substs.GRADLE_MAVEN_REPOSITORIES.each { repository ->
-            maven {
-                url = repository
-                if (gradle.mozconfig.substs.ALLOW_INSECURE_GRADLE_REPOSITORIES) {
-                    allowInsecureProtocol = true
-                }
-            }
-        }
+        mavenLocal()
+        maven { url "https://plugins.gradle.org/m2/" }
+        google()
 
-        maven {
-            url = "${gradle.mozconfig.topobjdir}/gradle/maven"
-        }
     }
 
     tasks.withType(KotlinCompile).configureEach {
diff -r c3bba5162c98 mobile/android/fenix/plugins/apksize/build.gradle
--- a/mobile/android/fenix/plugins/apksize/build.gradle	Mon Apr 21 16:33:21 2025 +0000
+++ b/mobile/android/fenix/plugins/apksize/build.gradle	Wed Apr 23 09:00:07 2025 +0000
@@ -9,14 +9,9 @@
 buildDir = "${gradle.mozconfig.topobjdir}/gradle/build/mobile/android/fenix/plugins/apksize"
 
 repositories {
-        gradle.mozconfig.substs.GRADLE_MAVEN_REPOSITORIES.each { repository ->
-            maven {
-                url = repository
-                if (gradle.mozconfig.substs.ALLOW_INSECURE_GRADLE_REPOSITORIES) {
-                    allowInsecureProtocol = true
-                }
-            }
-        }
+    mavenLocal()
+    maven { url "https://plugins.gradle.org/m2/" }
+    google()
 }
 
 dependencies {
diff -r c3bba5162c98 mobile/android/fenix/plugins/apksize/settings.gradle
--- a/mobile/android/fenix/plugins/apksize/settings.gradle	Mon Apr 21 16:33:21 2025 +0000
+++ b/mobile/android/fenix/plugins/apksize/settings.gradle	Wed Apr 23 09:00:07 2025 +0000
@@ -11,14 +11,9 @@
     }
 
     repositories {
-        gradle.mozconfig.substs.GRADLE_MAVEN_REPOSITORIES.each { repository ->
-            maven {
-                url = repository
-                if (gradle.mozconfig.substs.ALLOW_INSECURE_GRADLE_REPOSITORIES) {
-                    allowInsecureProtocol = true
-                }
-            }
-        }
+        mavenLocal()
+        maven { url "https://plugins.gradle.org/m2/" }
+        google()
     }
 }
 
diff -r c3bba5162c98 mobile/android/fenix/settings.gradle
--- a/mobile/android/fenix/settings.gradle	Mon Apr 21 16:33:21 2025 +0000
+++ b/mobile/android/fenix/settings.gradle	Wed Apr 23 09:00:07 2025 +0000
@@ -10,14 +10,9 @@
     }
 
     repositories {
-        gradle.mozconfig.substs.GRADLE_MAVEN_REPOSITORIES.each { repository ->
-            maven {
-                url = repository
-                if (gradle.mozconfig.substs.ALLOW_INSECURE_GRADLE_REPOSITORIES) {
-                    allowInsecureProtocol = true
-                }
-            }
-        }
+        mavenLocal()
+        maven { url "https://plugins.gradle.org/m2/" }
+        google()
     }
 
     includeBuild("../android-components/plugins/publicsuffixlist")
diff -r c3bba5162c98 mobile/android/focus-android/build.gradle
--- a/mobile/android/focus-android/build.gradle	Mon Apr 21 16:33:21 2025 +0000
+++ b/mobile/android/focus-android/build.gradle	Wed Apr 23 09:00:07 2025 +0000
@@ -7,14 +7,9 @@
 
 buildscript {
     repositories {
-        gradle.mozconfig.substs.GRADLE_MAVEN_REPOSITORIES.each { repository ->
-            maven {
-                url = repository
-                if (gradle.mozconfig.substs.ALLOW_INSECURE_GRADLE_REPOSITORIES) {
-                    allowInsecureProtocol = true
-                }
-            }
-        }
+        mavenLocal()
+        maven { url "https://plugins.gradle.org/m2/" }
+        google()
     }
 
     dependencies {
@@ -76,18 +71,10 @@
 
 allprojects {
     repositories {
-        gradle.mozconfig.substs.GRADLE_MAVEN_REPOSITORIES.each { repository ->
-            maven {
-                url = repository
-                if (gradle.mozconfig.substs.ALLOW_INSECURE_GRADLE_REPOSITORIES) {
-                    allowInsecureProtocol = true
-                }
-            }
-        }
+        mavenLocal()
+        maven { url "https://plugins.gradle.org/m2/" }
+        google()
 
-        maven {
-            url = "${gradle.mozconfig.topobjdir}/gradle/maven"
-        }
     }
 
     tasks.withType(KotlinCompile).configureEach {
diff -r c3bba5162c98 mobile/android/focus-android/settings.gradle
--- a/mobile/android/focus-android/settings.gradle	Mon Apr 21 16:33:21 2025 +0000
+++ b/mobile/android/focus-android/settings.gradle	Wed Apr 23 09:00:07 2025 +0000
@@ -10,14 +10,9 @@
     }
 
     repositories {
-        gradle.mozconfig.substs.GRADLE_MAVEN_REPOSITORIES.each { repository ->
-            maven {
-                url = repository
-                if (gradle.mozconfig.substs.ALLOW_INSECURE_GRADLE_REPOSITORIES) {
-                    allowInsecureProtocol = true
-                }
-            }
-        }
+        mavenLocal()
+        maven { url "https://plugins.gradle.org/m2/" }
+        google()
     }
 
     includeBuild("../android-components/plugins/publicsuffixlist")
diff -r c3bba5162c98 mobile/android/shared-settings.gradle
--- a/mobile/android/shared-settings.gradle	Mon Apr 21 16:33:21 2025 +0000
+++ b/mobile/android/shared-settings.gradle	Wed Apr 23 09:00:07 2025 +0000
@@ -12,14 +12,9 @@
     }
 
     repositories {
-        gradle.mozconfig.substs.GRADLE_MAVEN_REPOSITORIES.each { repository ->
-            maven {
-                url = repository
-                if (gradle.mozconfig.substs.ALLOW_INSECURE_GRADLE_REPOSITORIES) {
-                    allowInsecureProtocol = true
-                }
-            }
-        }
+        mavenLocal()
+        maven { url "https://plugins.gradle.org/m2/" }
+        google()
     }
 
     dependencies {
diff -r c3bba5162c98 settings.gradle
--- a/settings.gradle	Mon Apr 21 16:33:21 2025 +0000
+++ b/settings.gradle	Wed Apr 23 09:00:07 2025 +0000
@@ -9,14 +9,9 @@
     }
 
     repositories {
-        gradle.mozconfig.substs.GRADLE_MAVEN_REPOSITORIES.each { repository ->
-            maven {
-                url = repository
-                if (gradle.mozconfig.substs.ALLOW_INSECURE_GRADLE_REPOSITORIES) {
-                    allowInsecureProtocol = true
-                }
-            }
-        }
+        mavenLocal()
+        maven { url "https://plugins.gradle.org/m2/" }
+        google()
     }
 
     includeBuild("${rootProject.projectDir.absolutePath}/mobile/android/android-components/plugins/config")
